src/document/DocumentCommandHandlers.js

```diff
@@ -158,11 +166,13 @@ define(function (require, exports, module) {
         var currentDoc          = DocumentManager.getCurrentDocument(),
             windowTitle         = brackets.config.app_title,
             currentlyViewedFile = MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE),
-            currentlyViewedPath = currentlyViewedFile && currentlyViewedFile.fullPath,
-            readOnlyString      = (currentlyViewedFile && currentlyViewedFile.readOnly) ? "[Read Only] - " : "";
```

Comment:
You removed the change after resolving the conflict

Reply:
You mean the change to the file header? Which change was removed if otherwise please? I'm having trouble spotting the difference. Sorry if this is a newb question.